.github/CODEOWNERS

```diff
@@ -14,6 +14,13 @@
 # /scripts/
 # /tools/
 
+###########
```

Comment:
Why are we movving these as opposed to just removing the ** paths?

Reply:
Fixed in:
- https://github.com/Azure/azure-sdk-for-python/pull/28622